model/godleyTableWindow.cc

```diff
@@ -201,7 +201,14 @@ namespace minsky
           {
             CairoSave cs(surface->cairo());
             cairo_move_to(surface->cairo(), x, columnButtonsOffset);
-            colWidgets[col].draw(surface->cairo());
+            // Prevent the existence of more than one equity column in single equity column mode
+            if (godleyIcon->table._assetClass(col)==GodleyAssetClass::equity && col!=godleyIcon->table.cols()-1 && !godleyIcon->table.multipleEquities) {
+               godleyIcon->table.deleteCol(col);
+               colWidgets[col].draw(surface->cairo());
+		   }
+            else {
```

Comment:
I'm not quite sure what you're doing here, but seems unrelated to #1162. Could you please remove this and commit in a separate PR if so.

Reply:
Yes, this somehow snuck in to this PR, even after I had pulled from remote/master?. Sorry, I'll remove it right away, it is part of the multiple equity column PR.